sklearn/utils/multiclass.py

```diff
@@ -226,7 +226,7 @@ def check_classification_targets(y):
         )
 
 
-def type_of_target(y, input_name=""):
+def type_of_target(y, input_name="", raise_unknown=False):
```

Comment:
I'm tempted to have this always raise. Adding it since we expect estimators to raise an error message, but we don't really have a tool that raises that for the devs.